code/extensions/TimezoneMemberExtension.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+ * Class TimezoneMemberExtension.
+ *
+ * Adds all available timezones as an optional field to SilverStripe {@link Member}.
+ */
+class TimezoneMemberExtension extends DataExtension
+{
+    private static $db = [
+        'Timezone' => 'Varchar(255)',
+    ];
+
+    public function getTimezones() {
+        $timezones = timezone_identifiers_list();
+        return array_combine($timezones, $timezones);
+    }
+
+    public function updateCMSFields(FieldList $fields)
+    {   
```

Comment:
brace should be pushed up to the same line as `function`, e.g. `function foo($param) {`